Data/Customer.cs

```diff
@@ -1,6 +1,8 @@
 public class Customer
 {
     public int CustomerId { get; set; }
-    public string FirstName { get; set; }
-    public string LastName { get; set; }
-}
\ No newline at end of file
+
+    public string Email { get; set; }
+
+    public string Code { get; set; }
```

Comment:
ik denk dat code niet bij user hoort maar puur bij reservation?

Reply:
Maar dan zou de E-mail ook bij reservation horen toch? Aangezien we pas bij Reservation de Mail vragen. Wat zouden we eigenlijk kunnen doen met de Customer class.

Reply:
ja email hoort ook bij reservation customer class is om email te onthouden voor toekomstige mailing list denk ik

Reply:
Ik heb een tweede commit gemaakt en daar de Code bij Reservation.cs gezet is dat ok zo?

Reply:
oke looks good